eclipse-theia/theia
# Align available locales to VS Code

#### What it does

Aligns the languages we offer for translation to the official language packs offered by VS Code. See also:
https://code.visualstudio.com/docs/getstarted/locales#_available-locales

In detail:
* Add Korean, Turkish and Traditional Chinese to our default localizations
* Remove `pt-pt` from our offered localizations

#### How to test

Ensure that the localizations I've commited work as expected. They were produced by running the `translation-update.js` script.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
